libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testCollectionEditMessage.json

```diff
@@ -0,0 +1,29 @@
+{
+  "formdata" : {
+    "type" : "org.finos.symphony.toolkit.workflow.fixture.person",
+    "names" : [ "abc", "pqr" ],
+    "addresses" : [ {
+      "city" : "Pune"
```

Comment:
This isn't testing a primitive?  Don't we already have a test for beans like Address?

Reply:
Yes we have the test for beans like address. But we are checking is there any impact of beans with adding support for primitive. So this work class contains List<String> and List<Address>. I don't think so there is any issue with testing beans along with primitive